sidechain/consensus/aura/src/lib.rs

```diff
@@ -145,9 +146,14 @@ where
 	fn epoch_data(
 		&self,
 		header: &ParentchainBlock::Header,
+		shard: ShardIdentifierFor<Self::Output>,
```

Comment:
I think we are way too generic here. ShardIdentifier is one type only. everywhere. TBD

Reply:
I agree

Reply:
https://github.com/integritee-network/worker/issues/1411